packages/fast-components-styles-msft/src/carousel/index.ts

```diff
@@ -16,7 +16,7 @@ const styles: ComponentStyles<CarouselClassNameContract, DesignSystem> = (
     const white: string = "white";
     const darkGray: string = "#101010";
     const lightGray: string = "rgb(206, 206, 206)";
-    const lightGrayRest: string = "rgba(206, 206, 206, 0.6";
+    const lightGrayRest: string = "rgba(206, 206, 206, 0.6)";
```

Comment:
I already fixed this yesterday. 
https://github.com/Microsoft/fast-dna/pull/1496